CONTRIBUTING.md

```diff
@@ -29,9 +29,8 @@ This subfolder contains some example `.rst` files that show how to implement spe
 ------
 ## Run locally
 
-1. Set up a Python environment to work in, e.g.:
-   1. `python3 -m venv .venv/`
-   2. `source .venv/bin/activate`
-2. Run `pip install -r requirements-dev.txt`
-3. Run `pip install .` (must do this every time there is a change to the theme).
-4. Run `sphinx-build -b html docs/ docs/_build/html` to build the documentation. This will create the `/docs/_build` subfolder where you can see the html files generated based on the `.rst` files in the `docs` folder and the styles pulled from the `qiskit_sphinx_theme`.
+We use [Tox](https://tox.wiki/en/latest/), which you will need to install globally (e.g. using [`pipx`](https://pypa.github.io/pipx/)).
```

Comment:
I think we shouldn't replace the old instructions, there may be cases where people son't want to use tox for whatever reason. I think we should keep instructions for both methods

Reply:
With build processes, it's advantageous to consolidate around one documented way of doing things. It makes it easier to ensure that everyone is running the exact same commands, such as the options we set with Sphinx. One of the worst problems in "build engineering" / developer productivity is Works On My Machine™️ issues; using consistent tooling and commands helps to mitigate that.

It's also simpler for contributors to have only one documented way of doing things. Otherwise, people like the Digital team will need to make a decision of whether to use `venv` or use `tox` - that decision isn't really relevant to them. The want something that just works.

--

You're still welcome to use the `venv` approach rather than tox if you prefer (although I'd encourage migrating to Tox). This PR is only about what we document as the "blessed path".